### ISSUE TITLE
OOTDiffusion

### ISSUE THREAD

**Issue by De3dinthehe3d** [levihsu/OOTDiffusion#208]
**How to have a permanent hosting** 

_"/content
fatal: destination path 'OOTDiffusion' already exists and is not an empty directory.
/content/OOTDiffusion
aria2 is already the newest version (1.36.0-1).
0 upgraded, 0 newly installed, 0 to remove and 45 not upgraded.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
97edf5|OK  |       0B/s|/content/OOTDiffusion/images/demo.png

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
725a25|OK  |       0B/s|/content/OOTDiffusion/checkpoints/humanparsing/exp-schp-201908261155-lip.pth

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
e9df45|OK  |       0B/s|/content/OOTDiffusion/checkpoints/humanparsing/exp-schp-201908301523-atr.pth

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
33c372|OK  |       0B/s|/content/OOTDiffusion/checkpoints/openpose/ckpts/body_pose_model.pth

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
5bb097|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/feature_extractor/preprocessor_config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
9be828|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/ootd_hd/checkpoint-36000/unet_garm/config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
0ca13c|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/ootd_hd/checkpoint-36000/unet_garm/diffusion_pytorch_model.safetensors

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
bac8ab|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/ootd_hd/checkpoint-36000/unet_vton/config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
7aac19|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/ootd_hd/checkpoint-36000/unet_vton/diffusion_pytorch_model.safetensors

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
db28d1|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/scheduler/scheduler_config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
51633f|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/text_encoder/config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
e4ff58|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/text_encoder/pytorch_model.bin

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
b7e1e4|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/tokenizer/merges.txt

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
b59bd4|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/tokenizer/special_tokens_map.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
6eb2e6|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/tokenizer/tokenizer_config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
c23634|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/tokenizer/vocab.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
e72712|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/vae/config.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
4e09a3|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/vae/diffusion_pytorch_model.bin

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
88c57e|OK  |       0B/s|/content/OOTDiffusion/checkpoints/ootd/model_index.json

Status Legend:
(OK):download completed.

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
f55d51|OK  |       0B/s|/content/OOTDiffusion/checkpoints/clip-vit-large-patch14.tar

Status Legend:
(OK):download completed.
/content/OOTDiffusion/checkpoints
mkdir: cannot create directory ‘clip-vit-large-patch14’: File exists
/content/OOTDiffusion/checkpoints/clip-vit-large-patch14
./
./merges.txt
./preprocessor_config.json
./tf_model.h5
./.gitattributes
./vocab.json
./config.json
./model.safetensors
./tokenizer.json
./pytorch_model.bin
./tokenizer_config.json
./special_tokens_map.json
./flax_model.msgpack
./README.md
/content/OOTDiffusion/run
/usr/local/lib/python3.10/dist-packages/torch/utils/cpp_extension.py:1967: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(
2024-08-16 04:56:47.415683: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:485] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-08-16 04:56:47.456763: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:8454] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-08-16 04:56:47.473289: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1452] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-08-16 04:56:47.514753: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-08-16 04:56:49.848098: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/usr/local/lib/python3.10/dist-packages/diffusers/utils/outputs.py:63: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
/usr/local/lib/python3.10/dist-packages/diffusers/utils/outputs.py:63: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  torch.utils._pytree._register_pytree_node(
Loading pipeline components...: 100% 7/7 [00:01<00:00,  4.92it/s]
Loading pipeline components...: 100% 7/7 [00:00<00:00, 10.47it/s]
Running on local URL:  http:/.....................
Running on public URL: https:/..............

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)"_
